board/safety/safety_hyundai_common.h

```diff
@@ -19,11 +23,13 @@ enum {
 bool hyundai_ev_gas_signal = false;
 bool hyundai_hybrid_gas_signal = false;
 bool hyundai_longitudinal = false;
+int hyundai_max_steer = 0;
 uint8_t hyundai_last_button_interaction;  // button messages since the user pressed an enable button
 
 void hyundai_common_init(uint16_t param) {
   hyundai_ev_gas_signal = GET_FLAG(param, HYUNDAI_PARAM_EV_GAS);
   hyundai_hybrid_gas_signal = !hyundai_ev_gas_signal && GET_FLAG(param, HYUNDAI_PARAM_HYBRID_GAS);
+  hyundai_max_steer = param & HYUNDAI_PARAM_MAX_STEER;
```

Comment:
@adeebshihadeh do you want to create the steering safety struct on init, or modify it somehow?

Reply:
we have to be able to modify the max torque to whatever is passed in